pwiz_tools/Skyline/CommandLine.cs

```diff
@@ -3149,6 +3155,57 @@ public bool ExportSpecLib(string specLibFile)
             return true;
         }
 
+        /// <summary>
+        /// Export mProphet features as a .csv file
+        /// </summary>
+        /// <param name="mProphetFile">File path to export the mProphet file to</param>
+        /// <param name="targetPeptidesOnly">Do not include decoys, only targets</param>
+        /// <param name="bestOnly">Export best scoring peaks only</param>
+        /// <param name="excludeScores">A list of features to exclude from the exported file</param>
+        /// <returns>True upon successful import, false upon error</returns>
+        public bool ExportMProphetFeatures(string mProphetFile, bool targetPeptidesOnly, bool bestOnly, FeatureCalculators excludeScores)
+        {
+            // Excluding any scores requested by the caller
+            var calcs = new FeatureCalculators(PeakFeatureCalculator.Calculators.Where(c => excludeScores.IndexOf(c) < 0));

```

Comment:
Do this after the error checks and it could be inside the try block right above where "calcs" is used.